pages/common/stow.md

```diff
@@ -0,0 +1,24 @@
+# stow
+
+> Symlink manager.
+> Often used to manage dotfiles.
+
+- Symlink all files recursively to {{target}}:
```

Comment:
From the docs, I see that target is a directory right ? So shall we change target to "path/to/directory" ?

Reply:
Yep, thx!